### PR TITLE
enable favicon

### PR DESCRIPTION
**Pull request by rebornix**
enable favicon in index.js

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -1,14 +1,13 @@
 var express = require('express');
 var router = express.Router();
```
Comment (rebornix) [inline routes/index.js]: please remove.

Comment (rebornix) [inline routes/index.js]: remove.

Comment (auchenberg) [inline routes/index.js]: Logger used to be unsued. Please remove.

**Comment by auchenberg** [comment]
Overall LGTM, but tiny comment.

File: routes/index.js (hunk 1 above)
Comment (auchenberg) [inline routes/index.js]: Logger seems to be unused. Please remove.

**Comment by auchenberg** [comment]
Overall LGTM. One little nit.

File: routes/index.js [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
+var logger = require('morgan');
```
Comment (rebornix) [inline routes/index.js]: remove

File: routes/index.js (hunk 1 above)
Comment (auchenberg) [reply routes/index.js]: sf
sdf
df

Comment (chrisdias) [reply routes/index.js]: **this is a great PR**

File: app.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ hbs.registerPartials(__dirname + '/views/partials');
 app.set('view engine', 'hbs');
 
 // uncomment after placing your favicon in /public
-//app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
+app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
```
Comment (RMacfarlane) [inline app.js]: Remember to also check in the .ico file

File: routes/index.js (hunk 2 above)
Comment (RMacfarlane) [inline routes/index.js]: Remove?

File: routes/index.js [hunk 4]
```diff
@@ -1,14 +1,13 @@
...
 
 var fs = require('fs');
 
 var Cart = require('../models/cart');
```
Comment (RMacfarlane) [inline routes/index.js]: normalize casing

File: routes/index.js [hunk 5]
```diff
@@ -1,14 +1,13 @@
...
 var products = JSON.parse(fs.readFileSync('./data/products.json', 'utf8'));
 
 router.get('/', function (req, res, next) {
```
Comment (RMacfarlane) [inline routes/index.js]: new comment

**Pull request closed without merging**